addons/account/models/chart_template.py

```diff
@@ -259,6 +259,8 @@ def tax_template_changed(tax, template):
                     if rec:
                         for fname in x2manyfields:
                             for i, (line, vals) in enumerate(zip(rec[fname], values[fname])):
+                                if not isinstance(vals[2], dict):
+                                    continue
```

Comment:
Why is this an int, why are we expecting a dict?
Fix bugs, not symptoms.

Reply:
> Why is this an int, why are we expecting a dict? Fix bugs, not symptoms.

Hello,

Can you please check the updated changes?

Thank You!

Reply:
> Why is this an int, why are we expecting a dict? Fix bugs, not symptoms.

Hello @william-andre 

Can you please check the updated changes?

Reply:
> Why is this an int, why are we expecting a dict? Fix bugs, not symptoms.

Could you provide an answer to my question?

Reply:
> Why is this an int, why are we expecting a dict? Fix bugs, not symptoms.

Could you provide an answer to my question?

Reply:
Hello,

While updating the journal when reloading the template of 'Saudi and UAE' we are getting values as 0 from this line:-  https://github.com/odoo/odoo/blob/saas-16.2/addons/account/models/chart_template.py#L373

Below are the links the 'Saudi and UAE' template links for your reference!

1) `l10n_sa` :- https://github.com/odoo/odoo/blob/saas-16.2/addons/l10n_sa/models/template_sa.py#L63
2) `l10n_ae`:- https://github.com/odoo/odoo/blob/saas-16.2/addons/l10n_ae/models/template_ae.py#L58
3) `fields.py`:- https://github.com/odoo/odoo/blob/saas-16.2/odoo/fields.py#L3979